frontend/packages/sdk/src/create.ts

```diff
@@ -80,7 +81,7 @@ export interface CreateFnOptions {
 export interface CreateFnResult {
   api: ApiPromise;
   sidevmQuery: SidevmQuery;
-  instantiate: SidevmQuery;
+  instantiate: SidevmQuery<ContractInstantiateResult | InkQueryError>;
```

Comment:
Should we rename `SidevmQuery` to `RawQuery` and alias it as `SidevmQuery`? To me `instantiate()` is not a query to sidevm.

Reply:
IMO old codes will not continue updates and leave it there for compatible reason. For TypeScript project type declaration update may break downstream's linting.